### PR TITLE
CV node filter tooltips

### PR DESCRIPTION
**Pull request by SudheendraKatte**

### CODE REVIEW

File: src/Tooltip/TooltipDictionary.yaml [hunk 1]
```diff
@@ -8457,3 +8457,9 @@ stepGitlabAuth_apiUrl:
 pipelineInputStage:
    content: CI Stage can be skipped with New Artifact/Manifest Trigger using selective stage configuration.
    width: "400"
+controlNodeRegexFilterInput:
+   content: Refine RegEx expression to filter control (baseline) Nodes by PodName.
```
Comment (SudheendraKatte) [inline src/Tooltip/TooltipDictionary.yaml:8461]:
```suggestion
   content: RegEx expression to filter control (baseline) nodes by PodName.
```

**Review by praneshg239 — APPROVED** [review]

File: src/Tooltip/TooltipDictionary.yaml [hunk 2]
```diff
@@ -8457,3 +8457,9 @@ stepGitlabAuth_apiUrl:
 pipelineInputStage:
    content: CI Stage can be skipped with New Artifact/Manifest Trigger using selective stage configuration.
    width: "400"
+controlNodeRegexFilterInput:
+   content: RegEx expression to filter control (baseline) nodes by PodName.
+   width: "400"
+testNodeRegexFilterInput:
+   content: RegEx Expression to filter test (new) nodes by PodName.
```
Comment (brian-f-harness) [inline src/Tooltip/TooltipDictionary.yaml:8464]:
```suggestion
   content: RegEx expression to filter test (new) nodes by PodName.
```

File: src/Tooltip/TooltipDictionary.yaml [hunk 3]
```diff
@@ -8457,3 +8457,9 @@ stepGitlabAuth_apiUrl:
...
+   width: "400"
```
Comment (brian-f-harness) [inline src/Tooltip/TooltipDictionary.yaml:8465]: Could also consider using just RegEx and dropping _expression_.

**Review by brian-f-harness — APPROVED** [review]
LGTM

File: src/Tooltip/TooltipDictionary.yaml [hunk 4]
```diff
@@ -8457,3 +8457,9 @@ stepGitlabAuth_apiUrl:
 pipelineInputStage:
    content: CI Stage can be skipped with New Artifact/Manifest Trigger using selective stage configuration.
    width: "400"
+controlNodeRegexFilterInput:
+   content: RegEx expression to filter control (baseline) nodes by PodName.
+   width: "400"
+testNodeRegexFilterInput:
+   content: RegEx expression to filter test (new) nodes by PodName.
```
Comment (SudheendraKatte) [inline src/Tooltip/TooltipDictionary.yaml:8464]:
```suggestion
   content: RegEx to filter test (new) nodes by PodName.
```

File: src/Tooltip/TooltipDictionary.yaml [hunk 5]
```diff
@@ -8457,3 +8457,9 @@ stepGitlabAuth_apiUrl:
 pipelineInputStage:
    content: CI Stage can be skipped with New Artifact/Manifest Trigger using selective stage configuration.
    width: "400"
+controlNodeRegexFilterInput:
+   content: RegEx expression to filter control (baseline) nodes by PodName.
```
Comment (SudheendraKatte) [inline src/Tooltip/TooltipDictionary.yaml:8461]:
```suggestion
   content: RegEx to filter control (baseline) nodes by PodName.
```

**Pull request merged**